src/Sylius/Bundle/CoreBundle/Form/Type/ProductReviewAdminType.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Form\Type;
+
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+/**
+ * @author Daniel Richter <nexyz9@gmail.com>
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+class ProductReviewAdminType extends ProductReviewType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        parent::buildForm($builder, $options);
+
+        $builder
+            ->add('reviewSubject', 'entity', array(
+                'class'    => 'Sylius\Component\Core\Model\Product',
```

Comment:
We should use ``sylius_product_choice`` type here. Can be easily generated for you if you add ``choice`` under ``product.form`` in DI configuration.